storage/innobase/row/row0import.cc

```diff
@@ -610,7 +610,7 @@ class AbstractCallback
 
 		if (m_xdes != 0) {
 			const xdes_t*	xdesc = xdes(page_no, m_xdes);
-			ulint		pos = page_no % FSP_EXTENT_SIZE;
+			uint32_t	pos = page_no % FSP_EXTENT_SIZE;
```

Comment:
We could cache the value of `FSP_EXTENT_SIZE` in a data member `AbstractCallback::extent_size`. But, since the `ALTER TABLE…IMPORT TABLESPACE` functionality is rarely used, I would not expand the scope of this fix with that.